src/main/java/org/openstreetmap/atlas/geography/atlas/change/FeatureChange.java

```diff
@@ -482,12 +497,12 @@ else if (format == PrettifyStringFormat.MINIMAL_MULTI_LINE)
         builder.append(separator);
         builder.append("bounds=" + this.bounds() + ", ");
         builder.append(separator);
-        builder.append("afterView="
+        builder.append("afView="
                 + ((CompleteEntity<?>) this.afterView).prettify(completeEntityFormat) + ", ");
         builder.append(separator);
         if (this.beforeView != null)
         {
-            builder.append("beforeView="
+            builder.append("bfView="
```

Comment:
is that an abbreviation??

Reply:
Haha yes. It's a strategic abbreviation so that the actual contents of the `beforeView` and `afterView` are aligned on the same column. It makes it much easier to see the difference when quickly scanning the output.